### PR TITLE
docs: add static code margin-bottom

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

Fixes a small regression with no white space after static code blocks.

### CODE REVIEW

**Comment by burakukula** [comment]
hmmm, are we sure we want to have a bigger space than this?

Screenshot 2022-01-12 at 11 55 17

**Comment by suevalov** [comment]
@burakukula this PR is about bottom margin

This is production now: 

Screenshot 2022-01-12 at 11 56 16

**Comment by burakukula** [comment]
@suevalov yeah, it's not enough space for those cases, you are right.

**Review by burakukula — APPROVED** [review]

File: packages/website/components/LiveEditor/StaticSource.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ const styles = {
   root: css({
     padding: tokens.spacingM,
     borderRadius: tokens.borderRadiusMedium,
+    marginBottom: tokens.spacingL,
```
Comment (gui-santos) [inline packages/website/components/LiveEditor/StaticSource.tsx:14]:
```suggestion
    marginBottom: tokens.spacingM,
```

looking the picture you showed,
I think it should have the same marginBottom as a paragraph

**Review by gui-santos — CHANGES_REQUESTED** [review]

**Review by gui-santos — APPROVED** [review]

**Pull request merged**